http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/internal/conn/TlsProtocol.java

```diff
@@ -15,17 +15,20 @@
 
 package software.amazon.awssdk.http.apache.internal.conn;
 
+import software.amazon.awssdk.annotations.SdkInternalApi;
+
 /**
  * TLS protocols arranged in descending order of security preference in terms of
  * their ordinal numbers. See <a href=
  * "http://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#jssenames"
  * >JSSE Standard Names</a>.
  */
+@SdkInternalApi
 enum TlsProtocol {
-    TLSv1_2("TLSv1.2"),
+    TLS_V1_2("TLSv1.2"),
     // most secure/preferred
```

Comment:
Are these comments to the preceding line?

Reply:
No idea.